assets/javascripts/discourse/components/ai-llm-quota-editor.gjs

```diff
@@ -1,178 +0,0 @@
-import Component from "@glimmer/component";
```

Comment:
I just moved all of this in the form, I could split it in multiple components, but there was not much value to it. It's implemented using the collection from form kit: https://meta.discourse.org/t/discourse-toolkit-to-render-forms/326439?silent=true